charts/rpe-expressjs-template/values.yaml

```diff
@@ -3,4 +3,8 @@ nodejs:
   aadIdentityName: rpe
   ingressHost: rpe-expressjs-template.{{ .Values.global.environment }}.platform.hmcts.net
   image: 'hmctspublic.azurecr.io/rpe/expressjs-template:latest'
+  keyVaults:
```

Comment:
Should we comment this section by default ? and mention in readme/ comments ? I am conscious that adding it by default means they cannot get it past preview till they create the vault, appinsights.